src/main/java/com/hubspot/jinjava/loader/RelativePathResolver.java

```diff
@@ -0,0 +1,26 @@
+package com.hubspot.jinjava.loader;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import com.hubspot.jinjava.interpret.JinjavaInterpreter;
+
+public class RelativePathResolver implements LocationResolver {
+  public static final String CURRENT_PATH_CONTEXT_KEY = "current_path";
+
+  @Override
+  public String resolve(String path, JinjavaInterpreter interpreter) {
+    if (path.startsWith("./")) {
+      String parentPath = interpreter.getContext().getCurrentPathStack().peek()
+          .orElseGet(() -> (String) interpreter.getContext().getOrDefault(CURRENT_PATH_CONTEXT_KEY, ""));
+
+      Path templatePath = Paths.get(parentPath);
+      Path folderPath = templatePath.getParent() != null ? templatePath.getParent() : Paths.get("");
+      if (folderPath != null) {
```

Comment:
I don't think that `folderPath` would ever be null, but the build was failing because `Travis CI` was saying there was a possible null dereference. 🤷‍♂